debian/build.py

```diff
@@ -46,11 +46,11 @@ def main():
   packages = {
     'kubectl': (
       ('1.3.7', '00'),
-      ('1.4.0-beta.3', '02'),
+      ('1.4.0-beta.5', '02'),
```

Comment:
We can use it to release multiple versions of the same release in case of crisis. It's an escape hatch.